cluster/terraform_kubernetes/config/filebeat/filebeat.yml.tmpl

```diff
@@ -14,6 +14,25 @@ filebeat.autodiscover:
             enabled: false
           deployment: true
 
+# Global processors run AFTER enrichment and after all inputs
+processors:
+  - drop_fields:
+      fields:
+      - agent.ephemeral_id
+      - agent.id
+      - agent.name
+      - agent.version
+      - app.duration
+      - app.duration_ms
+      - app.pid
+      - app.tags
+      - app.thread
+      - ecs.version
+      - container.runtime
+      - kubernetes.pod.uid
+      - kubernetes.namespace_uid
```

Comment:
do we need kubernetes.labels.app, kubernetes.labels.pod-template-hash, kubernetes.namespace_labels.kubernetes_io/metadata_name, or anything below kubernetes.labels